core/src/test/java/io/questdb/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -1910,6 +1913,74 @@ public void testGeoHashSelectStr() throws Exception {
         testGeoHashSelect(false, false);
```

Comment:
Could you also test : 
insert 
insert as select 
truncate 
alter table 
create table 
vacuum
repair table 
snapshot database 
backup 
?

Reply:
all done